platform/Stream.h

```diff
@@ -58,22 +59,23 @@ class Stream : public FileLike, private NonCopyable<Stream> {
     {
         return _file;
     }
-
-protected:
-    virtual int close();
+#endif // !MBED_CONF_PLATFORM_STDIO_MINIMAL_CONSOLE_ONLY
     virtual ssize_t write(const void *buffer, size_t length);
     virtual ssize_t read(void *buffer, size_t length);
+    virtual int close();
```

Comment:
Pointless reordering - possible binary incompatibility with minimal console off. (On would be incompatible anyway).

Reply:
That was unintended will return as it was. Thanks.